snprc_ehr/resources/queries/snprc_ehr/ExportSndLookups.sql

```diff
@@ -4,7 +4,7 @@ SELECT
     Value as VALUE,
     CASE
         WHEN l.Displayable = 0 THEN 'Y'
-        WHEN l.Displayable = 1 THEN ''
+        WHEN l.Displayable = 1 THEN 'N'
```

Comment:
The source data is a boolean - do you really need to check both cases? How about: 
`CASE WHEN l.Displayable = 0 THEN 'Y' ELSE 'N'  END AS IS_HIDDEN`